### PR TITLE
Added Find Total Number of Connected Component in Graph

### PR DESCRIPTION
**Pull request by hardik-dadhich**
Issue: https://github.com/keon/algorithms/issues/641
1) Added `count_connected_number_of_component.py` file in `Algorithm /graph` section.

### CODE REVIEW

**Comment by ericklarac** [comment]
@hardik-dadhich Nice algorithm. 

Could you please add unit tests and the algorithm on the corresponding section on the readme? 

You can check out the following pull request as a guide 😄 
https://github.com/keon/algorithms/pull/631/files

**Comment by hardik-dadhich** [comment]
Sure, @ericklarac 
will update the PR soon!

File: tests/test_count_connected_components.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#import modules
+import unittest
+from algorithms.graph import count_connected_number_of_component
+
+class TestConnectedComponentInGraph(unittest.TestCase):
+	""" Class to performs unit tests """
+	
+	def test_count_connected_components(self):
```
Comment (ericklarac) [inline tests/test_count_connected_components.py]: @hardik-dadhich add this code to the ```tests/test_graph.py``` file.

File: tests/test_count_connected_components.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	            |
+	            |
+	            4
+	            
+	            output = 3
+
+		"""
+		expected_result = 3
+		
+		# adjacency list representation of graph
```
Comment (ericklarac) [inline tests/test_count_connected_components.py]: Could you please add more tests? I mean, empty graph as input, or a set of nodes without edges, to increase the test cases coverage.

Comment (hardik-dadhich) [reply tests/test_count_connected_components.py]: sure! 
on the way

**Comment by hardik-dadhich** [comment]
@ericklarac updated PR

File: tests/test_graph.py [hunk 3]
```diff
@@ -208,4 +208,56 @@ def test_bellman_ford(self):
             'e': {'a': 7, 'b': 5, 'd': 1}
         } 
     
-        self.assertEqual(True, bellman_ford(graph2, 'a'))
```
Comment (ericklarac) [inline tests/test_graph.py]: @hardik-dadhich Just keep this line please, to keep both tests in ```bellman_ford``` class. 😄

**Pull request merged**